_zh-cn/tour/multiple-parameter-lists.md

```diff
@@ -13,3 +13,71 @@ language: zh-cn
 next-page: case-classes
 previous-page: nested-functions
 ---
+
+方法可以定义多个参数列表，当使用较少的参数列表调用多参数列表的方法时，会产生一个新的函数，该函数接收剩余的参数列表作为其参数。这被称为[柯里化](https://zh.wikipedia.org/wiki/%E6%9F%AF%E9%87%8C%E5%8C%96)。
+
+下面是一个例子，在Scala集合中定义的特质[Traversable](/zh-cn/overviews/collections/trait-traversable.html)：
+
+```
+def foldLeft[B](z: B)(op: (B, A) => B): B
+```
+
+`foldLeft`从左到右，以此将一个二元运算`op`应用到初始值`z`和该迭代器（traversable)的所有元素上。以下是该函数的一个用例：
+
+从初值0开始, 这里 `foldLeft` 将函数 `(m, n) => m + n` 依次应用到列表中的每一个元素和之前累积的值上。
+
+```tut
+val numbers = List(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
+val res = numbers.foldLeft(0)((m, n) => m + n)
+print(res) // 55
+```
+
+多参数列表有更复杂的调用语法，因此应该谨慎使用，建议的使用场景包括：
+
+#### 单一的函数参数
+   在某些情况下存在单一的函数参数时，例如上述例子`foldLeft`中的`op`，多参数列表可以使得传递匿名函数作为参数的语法更为简洁。如果不使用多参数列表，代码可能像这样：
+   
+```
+numbers.foldLeft(0, {(m: Int, n: Int) => m + n})
+```
+   
+   注意使用多参数列表时，我们还可以利用Scala的类型推断来让代码更加简洁（如下所示），而如果没有多参数列表，这是不可能的。
+    
+```
+numbers.foldLeft(0)(_ + _)
+```
+   上述声明`numbers.foldLeft(0)(_ + _)`让我们可以指定参数`z`，然后再传递折叠函数，并且可以复用，如下所示：
```

Comment:
```
-让我们可以指定参数`z`，然后再传递折叠函数，并且可以复用，如下所示：
+上述语句让我们可以固定参数`z`来形成一个部分函数进行传递，达到复用的目的，如下所示：
```

Reply:
After doing some research<sup>[1]</sup><sup>[2]</sup>, I kind of believe that's what's supposed to mean, but not 100 percent sure.
<sup>1</sup> https://www.grokacademy.com/lessons/demystifying-currying-and-partial-function-application/
<sup>2</sup> https://goo.gl/oGhpTW

Reply:
I think all it wants to say is `currying allows you to fix z and fold the rest of the collection with the partially applied function`... `Above statement numbers.foldLeft(0)(_ + _)` in the Eng version is a bit confusing

Reply:
@dongxuwang I think you're right.

Reply:
@Qi77Qi Agree!
It always makes us crazy that partially applied function, partially function application, partial function. together with the PartialFunction etc.